From a6705653cf136f70c56d1f897b25a0aa481ba755 Mon Sep 17 00:00:00 2001
From: Bert van Hall <bert.vanhall@avionic-design.de>
Date: Fri, 20 Jan 2017 10:50:43 +0100
Subject: [PATCH 1/3] JSC/heap: fix symbol visibility

Mark Heap::heap as inline instead of ALWAYS_INLINE to avoid forcing
inlining and get it as an exported symbol in libjavascriptcore.so.

Signed-off-by: Bert van Hall <bert.vanhall@avionic-design.de>
---
 Source/JavaScriptCore/heap/HeapInlines.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/JavaScriptCore/heap/HeapInlines.h b/Source/JavaScriptCore/heap/HeapInlines.h
index 73d7859..8b3675f 100644
--- a/Source/JavaScriptCore/heap/HeapInlines.h
+++ b/Source/JavaScriptCore/heap/HeapInlines.h
@@ -63,7 +63,7 @@ inline bool Heap::isCollecting()
     return m_operationInProgress == FullCollection || m_operationInProgress == EdenCollection;
 }
 
-ALWAYS_INLINE Heap* Heap::heap(const HeapCell* cell)
+inline Heap* Heap::heap(const HeapCell* cell)
 {
     return cell->heap();
 }
-- 
2.7.4

